db/people.yml

```diff
@@ -2693,3 +2693,84 @@ zzgong:
   service: []
   skills: []
   title: Dr.
+wzhou:
+  active: false
+  aka:
+    - Wenduo Zhou
+    - Zhou, W.
+    - Zhou, Wenduo
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 1999
+      degree: PhD
+      institution: The University of Georgia
+      end_year: 2006
+  employment:
+    - begin_year: 2009
+      begin_month: 6
+      organization: Oak Ridge National Laboratory
+      position: Software Engineer
+    - begin_year: 2008
+      begin_month: 5
+      end_month: 6
+      end_year: 2009
+      group: bg
+      status: postdoc
+      advisor: sbillinge
+      organization: Columbia University
+      position: Assistant Research Scientist
+    - begin_year: 2006
+      begin_month: 9
+      end_month: 5
+      end_year: 2008
+      organization: Michigan State University
+      position: Assistant Research Scientist
+  name: Wenduo Zhou
+  position: Software Engineer
+asartbaeva:
+  active: false
+  aka:
+    - Asel Sartbaeva
+    - Sartbaeva, A.
+    - Sartbaeva, Asel
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 2002
+      degree: PhD in Mineral Physics
+      institution: University of Cambridge
+      end_year: 2004
+    - begin_year: 2001
+      degree: MPhil in Mineral Physics
+      institution: University of Cambridge
+      end_year: 2002
+    - begin_year: 1994
+      degree: MSc in Natural Science
+      institution: Kyrgyzskij Rossijskij Slavjanskij Universitet
+      end_year: 1999
+  employment:
```

Comment:
I think there should be an entry that shows her as a visitor in my group, no?

Reply:
@sbillinge I saw online that she was your Post Doctoral Associate at Arizona State University. Should I put that for the time period 2005-2007 or she was also in your group in Columbia?

Reply:
I was at Michigan State, not Arizona State.  She was never at Columbia.  Can you send me what it says online about her history?  Send a link maybe?  Sorry it is complicated...